forest/cli/tests/config_tests.rs

```diff
@@ -3,7 +3,9 @@
 use assert_cmd::Command;
 use forest_cli_shared::cli::{Client, Config};
 use rand::Rng;
+#[allow(unused_imports)]
 use std::{fs::read_dir, io::Write, net::SocketAddr, path::PathBuf, str::FromStr};
+#[allow(unused_imports)]
```

Comment:
can get rid of this, if we do `tempdir::TempDir` within `#[cfg(feature = "slow_tests")]`, likewise where applicable.

Reply:
fixed